src/background.js

```diff
@@ -87,6 +112,21 @@ chrome.tabs.onActivated.addListener(async function(activeInfo) {
   }
   cache.push(activeTab);
   await setMRU(cache);
+
+  // capture the visible tab
+  // store it?
+  let image = await chrome.tabs.captureVisibleTab();
```

Comment:
This keeps throwing an error when I switch tabs saying that "Tabs cannot be edited right now". I tried printing the tabImages array but it isn't printing anything.